frontend/sozisel-app/webpack.config.js

```diff
@@ -74,6 +76,7 @@ module.exports = {
     historyApiFallback: true
   },
   plugins: [
+    new FaviconsWebpackPlugin(path.join(__dirname, "public", "favicon.ico"),),
```

Comment:
Favicon is just a simple file, why do we have to use some fancy webpack plugin for that?

Reply:
it didn't work without it.